core/server/api/subscribers.js

```diff
@@ -275,7 +275,10 @@ subscribers = {
                 return Promise.reject(new errors.ValidationError(i18n.t('errors.api.db.selectFileToImport')));
             }
 
-            // TODO: check for valid entries
+            // Check for valid file type
+            if (!utils.checkFileIsValid(options, ['text/csv','application/csv'], ['csv'])) {
```

Comment:
Hey thanks for your PR. This line seems to be wrong, because importing a real CSV file doesn't work anymore.
I assume you need to use `['.csv']` instead of `['csv']`.